src/main/java/org/projecteka/hiu/Application.java

```diff
@@ -28,10 +31,19 @@ private static Connection openConnection() throws Exception {
         String jdbcUrl = System.getProperty("jdbc.url");
         String jdbcUsername = System.getProperty("jdbc.username");
         String jdbcUserPwd = System.getProperty("jdbc.password");
-        if (isEmptyString(jdbcUrl) || isEmptyString(jdbcUsername) || isEmptyString(jdbcUserPwd)) {
+        String jdbcDatabase = System.getProperty("jdbc.database");
+        if (isEmptyString(jdbcUrl) || isEmptyString(jdbcUsername) || isEmptyString(jdbcUserPwd) || isEmptyString(jdbcDatabase)) {
             throw new Exception("you must set jdbc.url, jdbc.username, jdbc.password as properties, either using -D option or setting as env");
         }
-        return DriverManager.getConnection(jdbcUrl, jdbcUsername, jdbcUserPwd);
+        Connection connection = DriverManager.getConnection(jdbcUrl, jdbcUsername, jdbcUserPwd);
+        Statement stmt = connection.createStatement( ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
+        ResultSet rs = stmt.executeQuery(String.format("SELECT FROM pg_database WHERE datname = '%s'", jdbcDatabase));
```

Comment:
Instead of constructing queries using "concatenation" its better to use PreparedStatement. See: https://cheatsheetseries.owasp.org/cheatsheets/SQL_Injection_Prevention_Cheat_Sheet.html